x/concentrated-liquidity/swaps_test.go

```diff
@@ -3535,10 +3535,10 @@ func (s *KeeperTestSuite) TestSwap_MinSpotPriceMigration() {
 		originalTick := pool.GetCurrentTick()
 
 		// esimate amount in to swap left all the way until the new min initialized tick
-		amountOneOut, _, _ := s.computeSwapAmountsInGivenOut(poolId, pool.GetCurrentSqrtPrice(), types.MinInitializedTickV2, true, false)
+		amountOneOut, _, _ := s.computeSwapAmountsInGivenOut(poolId, pool.GetCurrentSqrtPrice(), types.MinInitializedTick, true, false)
 
 		// estimate the amount in to fund
-		amountZeroIn, _, _ := s.computeSwapAmounts(poolId, pool.GetCurrentSqrtPrice(), types.MinInitializedTickV2, true, false)
+		amountZeroIn, _, _ := s.computeSwapAmounts(poolId, pool.GetCurrentSqrtPrice(), types.MinInitializedTick, true, false)
```

Comment:
Note: swaps to v2 min tick are not enabled anymore.

Previously the estimation would stop the current min tick anyways, so this request to estimate till `MinInitializedTickV2` is invalid at this time. To be re-added in a future PR.

Reply:
The definition for min initialized tick will be the same across all pools at this time though right? So we shouldn't need to name it something different if I am understanding correctly.

Reply:
Is this tracked in an issue?

Reply:
Already addressed here:
https://github.com/osmosis-labs/osmosis/pull/6380/files#r1325756769

Reply:
> The definition for min initialized tick will be the same across all pools at this time though right? So we shouldn't need to name it something different if I am understanding correctly.

See: https://github.com/osmosis-labs/osmosis/issues/6318